episodes/22-external.md

```diff
@@ -26,8 +20,40 @@ exercises: 25
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-::::::::::::::::::::::::::::::::::::::  challenge
+## Documenting your lesson
+
+For collaborative projects, it is important to spend the time on ‘external-facing’ features, such as documentation, to make the project as welcoming and as easy to get involved in as possible for newcomers who may not have a deep grasp of the project or know all its existing problems. Such documentation will be useful to yourself and other team members at well, e.g. if you are trying to come back to the project after a break or are reusing it for a new collaboration in the future.
+
+Your lesson documentation should contain the following information, which should be kept up to date.
+
+### Description of the project - `README` file
+
+`README.md` file (located in the root directory) is the first piece of documentation that people will land on when visiting your lesson repository. It should concisely explain what the lesson is about and who it is for, and contain links and pointers to further information, including:
+
+  - **rendered version of the lesson** - even though this is typically included in the `About` section of the repository, it is always useful to link to the URL where the rendered lesson is visible.
+  - **contact information** - include email addresses or mailing lists or other details on how to get in touch with the lesson maintainers.
+  - **contributing** - this is an opportunity to list what kinds of contribution are sought and how to get involved in lesson development for new contributors. You can provide more details in a separate `CONTRIBUTING.md` file within the repository’s root directory and link to it from `README`.
+  - **credits/acknowledgements** - make sure to credit those who have helped in the lesson's development or inspired it, and/or any resources/templates that you have reused. You can also link to a separate `AUTHORS.md` file within the repository’s root directory to list all the people who contributed to the lesson content (if this list starts to become too large to include in the `README` itself).
+  - **citation** - it’s a very good idea to specify a reference to an appropriate academic publication so others can cite the use of the lesson in their own publications and media. A common practice is to include the citation information in a separate `CITATION.md` file within the repository’s root directory and link to it from `README`.
```

Comment:
pinging @zkamvar for a review on this point: I know there is discussion of switching over to `CITATION.cff` format - is that something we need to consider/be aware of while drafting this curriculum?

Reply:
Good point, GitHub has adopted the .cff citation format. As an aside, many SSI's fellows and collaborators are involved in the [cff project](https://github.com/citation-file-format/citation-file-format).

Reply:
Addressed via f4fd959.